### DISCUSSION TITLE
Pagelock feature not supported in rights/roles configuration?

### DISCUSSION THREAD

**Discussion by mkjn72** [pimcore/pimcore#13650] [Q&A / Support] [answered]
I've been searching for a way to configure a role in Pimcore that will allow me to grant the 'lock page' or 'lock page and propagate to children' function to users. Without making them 'admin'.

![image](https://user-images.githubusercontent.com/68860845/203290499-b5e3066d-1a81-417e-aa41-50e2d08621f4.png)

Either I'm looking in the wrong place or this is not supported in Pimcore X (yet)? Any help/pointers appreciated

**Comment by mattamon** [comment, marked as answer]
Hey @mkjn72 the lock features are admin only.
There are no permissions bound to this.
